instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/config/internal/CommonConfig.java

```diff
@@ -56,6 +57,8 @@ public CommonConfig(InstrumentationConfig config) {
                 new ArrayList<>(HttpConstants.KNOWN_METHODS)));
     statementSanitizationEnabled =
         config.getBoolean("otel.instrumentation.common.db-statement-sanitizer.enabled", true);
+    sqlCommenterEnabled =
+        config.getBoolean("otel.instrumentation.common.db-sqlcommenter.enabled", false);
```

Comment:
I chose to use `sqlcommenter` instead of `sql-commenter` because https://google.github.io/sqlcommenter/ also spells it as one word.